Marlin/Marlin_main.cpp

```diff
@@ -5956,7 +5962,8 @@ inline void gcode_M17() {
           nozzle_timed_out |= thermalManager.is_heater_idle(e);
 
       #if ENABLED(ULTIPANEL)
-        if (nozzle_timed_out) ensure_safe_temperature();
+        if (nozzle_timed_out)
+          lcd_advanced_pause_show_message(ADVANCED_PAUSE_MESSAGE_CLICK_TO_HEAT_NOZZLE);
```

Comment:
As long as it works! I'll have a closer look to make sure I understand what's going on. At some point we'll want to clean up and standardize this LCD stuff more, so I should get a handle on it now!